lib/iex/lib/iex.ex

```diff
@@ -222,6 +222,16 @@ defmodule IEx do
       ...(1)> #iex:break
       ** (TokenMissingError) iex:1: incomplete expression
 
+  ## Exiting the shell
```

Comment:
@josevalim The problem is that it says "press Ctrl+C to exit", while `Ctrl+C` just opens the BREAK menu which does not explain how to exit. Pressing `Ctrl+C` twice is intuitive but annoying, and nothing tells about `Ctrl+\`.
I'd say the welcome text should tell "press Ctrl+C to open BREAK menu" or something similar, and the Break menu should list the `(q)uit` option which is not listed currently (I still don't understand where this is located in source files).